pkgs/by-name/ni/nixos-firewall-tool/package.nix

```
@@ -0,0 +1,15 @@
+{ writeShellApplication, iptables, lib, ... }:
```

```suggestion
{ writeShellApplication, iptables, lib }:
```

This is not a module declaration ;)